examples/tutorials/Introduction to Langrangian Neural Networks.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<p>Put this cell at the top of the notebook, near the importing of packages. You've used <code>.detach()</code> in above cells, it's better to keep the device constant at the start, instead of switching mid tutorial </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3905/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>